.cirrus.yml

```diff
@@ -3,13 +3,15 @@ gcp_credentials: ENCRYPTED[987a78af29b91ce8489594c9ab3fec21845bbe5ba68294b8f6def
 # LINUX
 task:
   gke_container:
-   image: gcr.io/flutter-cirrus/build-engine-image:latest
-   cluster_name: build-32-cluster
-   zone: us-central1-a
-   namespace: default
-   cpu: 30 # can't use all 30-cores; system pods needs cores too
-   memory: 100Gb # similarly, can't use all 100Gb memory
+    dockerfile: "ci/docker/build/Dockerfile"
+    builder_image_name: docker-builder # gce vm image
+    cluster_name: build-32-cluster
```

Comment:
The diff for the rest of these is because before they were only indented one space